lib/memcached/client.rb

```diff
@@ -10,17 +10,7 @@ class Client
     FLAGS = 0x0
 
     DEFAULTS = {
-      :hash => :fnv1_32,
-      :distribution => :consistent_ketama,
-      :ketama_weighted => true,
-      :retry_timeout => 60,
-      :timeout => 0.25,
-#      :poll_max_retries => 1, TODO: doesnt exist anymore
-      :connect_timeout => 0.25,
-      :hash_with_prefix_key => true,
-      :server_failure_limit => 2,
       :verify_key => true,
-      :io_key_prefetch => true,
     }
```

Comment:
I am ok with using all the defaults... would be good only to document the differences btwn the 1.0 defaults and the new ones..